docker-compose.yml

```diff
@@ -49,6 +50,29 @@ x-backend: &backend
       condition: service_healthy
 
 services:
+  devcontainer:
+    <<: *backend
+    environment:
+      MALLOC_ARENA_MAX: 2
+      WEB_CONCURRENCY: ${WEB_CONCURRENCY:-1}
+    volumes:
+      - ${LOCAL_WORKSPACE_FOLDER:-.}:/workspaces/forem:cached
+      - bundle:/usr/local/bundle
+      - cypress:/root/.cache/Cypress
+      - rails_cache:/workspaces/forem/tmp/cache
+      - assets:/workspaces/forem/public/assets
+      - node_modules:/workspaces/forem/node_modules
+      - packs:/workspaces/forem/public/packs
+      - packs-test:/workspaces/forem/public/packs-test
+      - history:/usr/local/hist
+      - ${LOCAL_WORKSPACE_FOLDER:-.}/.dockerdev/.psqlrc:/root/.psqlrc:ro
+      - /var/run/docker.sock:/var/run/docker-host.sock
+    command: sleep infinity
+    tmpfs:
+      - /workspaces/forem/tmp/pids
+    ports:
+      - '3000:3000'
```

Comment:
This exists specifically for devcontainer. It can live in a separate `docker-compose.devcontainer.yml` that we can extend, on the main `docker-compose`.